src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -458,17 +466,18 @@ object GermlineAssemblyCaller {
 
           possibleVariant
         })
-
       })
+
     variants.toSet
   }
 
   /**
-    * Call variants using a pileup and genotype likelihoods
-    * @param pileup Pileup at a given locus
-    * @return Possible set of called variants
-    */
-  def callPileupVariant(pileup: Pileup): Set[CalledAllele] = {
+   * Call variants using a pileup and genotype likelihoods
```

Comment:
Hm - it seems like intellij makes them one way and we want them the other? No preference either way, whatever is easiest really. I think scalariform would move these around too